docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -56,6 +56,15 @@
               </ul>
             </li>
             <li>
+              <a href="#" data-toggle="dropdown" class="dropdown-toggle">Security<b class="caret"></b></a>
+              <ul class="dropdown-menu">
+                <li><a href="{{BASE_PATH}}/security/overviwe.html">Overview</a></li>
```

Comment:
@prasadwagle Thx for your quick response!
I left some comments again in this PR just a moment ago.